src/components/search/Search.jsx

```diff
@@ -127,14 +139,29 @@ const Search = (props) => {
               <div className="input-group" style={{ width: '100%' }}>
                 <input id="searchInput" type="text" className="form-control"
                        onChange={ (event) => setQueryString(event.target.value) }
-                       onKeyPress={handleKeyPress} />
+                       onKeyPress={handleKeyPress}
+                       placeholder="Enter query string"
+                       value={ queryString } />
                 <span className="input-group-btn">
                   <button className="btn btn-default"
                           type="submit"
+                          title="Submit search"
                           aria-label="Submit search"
                           data-testid="Submit search">
                     <FontAwesomeIcon className="fa-search" icon={faSearch} />
                   </button>
+                  <button className="btn btn-default"
+                          type="button"
+                          aria-label="Clear query string"
+                          title="Clear query string"
+                          data-testid="Clear query string"
+                          onClick={() => {
+                            setQueryString('')
```

Comment:
Blank the query string in the input field, reset the URI in the dropdown, and ditch the search results.